### PR TITLE
Returns default AI behavior cooldown to .8 seconds

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
#60249 was a very nice (and sorta funnily numbered) PR that added a lot more flexibility to the AI datum system, but for whatever reason @ma44 effectively changed how often behaviors run from every 0.8 seconds to 0.1 seconds without actually accounting for how running eight times as often might affect existing AI. The harass dog AI behavior, for example, is currently able to crit someone with ~20 attacks in 2 seconds.

This sets the default behavior cooldown to the old behavior subsystem's wait of 0.8 seconds, which should hopefully return the old behavior and provide a saner default for the future. There might be side effects here with co-opting an existing var of action_cooldown, so reviews are welcome.

## Why It's Good For The Game
Behaviors running 10 times a second by default is kinda nuts and overkill

## Changelog
:cl: Ryll/Shaps
fix: Default AI behavior cooldowns have been returned back to 0,8 seconds instead of 0.1 seconds, meaning Ian will no longer disembowel you with 20 attacks in 2 seconds.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
caption (1)

**Comment by interestingusernam3** [comment]
unknown-15

File: code/datums/ai/_ai_behavior.dm [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	var/required_distance = 1
 	///Flags for extra behavior
 	var/behavior_flags = NONE
-	///Cooldown between actions performances
-	var/action_cooldown = 0
+	///Cooldown between actions performances, defaults to the value of CLICK_CD_MELEE because that seemed like a nice standard for the speed of AI behavior
```
Comment (Fikou) [inline code/datums/ai/_ai_behavior.dm:7]: if it defaults to CLICK_CD_MELEE why not just make the value the define CLICK_CD_MELEE

File: code/datums/ai/_ai_behavior.dm [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+	var/action_cooldown = 0.8 SECONDS
```
Comment (Mothblocks) [inline code/datums/ai/_ai_behavior.dm:8]:
```suggestion
	var/action_cooldown = CLICK_CD_MELEE
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by maxymax13** [comment]
buying dogs from cargo to pulverize antags

**Comment by AMonkeyThatCodes** [comment]
this is dumb, the change was fine just define your behaviors with proper vars.

**Comment by Ryll-Ryll** [comment]
None of the existing AI behaviors were written with running 10 times a second in mind, it's totally unnecessary

**Comment by AMonkeyThatCodes** [comment]
then set them to 0.8, if people are not setting bespoke values in their ai behavior it should be noticeable.

**Comment by AMonkeyThatCodes** [comment]
:suspect:

**Pull request merged**